tick/inference/base/learner_glm.py

```diff
@@ -53,21 +53,29 @@ class LearnerGLM(LearnerOptim):
         Record history information when ``n_iter`` (iteration number) is
         a multiple of ``record_every``
 
+    random_state : int seed, RandomState instance, or None (default)
```

Comment:
It should be moved to Other Parameters after `elastic_net_ratio` (it should be in the same order as in the constructor)